Add a usage guide for the scripts in the readme (#3)
MadLadSquad/make-me-a-hanzi-tool: README.md

# make-me-a-hanzi-tool
A (probably) working version of the make-me-a-hanzi tool.

Building:

> 1. Install mongodb
> 1. Install node and run `npm i`
> 1. Install meteor using the following command:

Reviewer: ```suggestion
1. Install mongodb
2. Install node and run `npm i`
3. Install meteor using the following command:
```

    3. R - Clear changes in the stroke editor

>     4. N - Next character

Reviewer: Missing alot of bindings. Also, i never heard of `N` before

Author: Well, `N` could be one of the following:
1. `n`
1. `j`
1. `k`

Colemak user moment :skull:

6. Note: When cutting 2 intersecting strokes, create 1 or 2 rectangles in the overlapped area when cutting. If any unneeded strokes appear deselect them

## Additional script usage for Youyin
In addition to improving the editor, we've also added scripts to make generating data for [youyin](https://github.com/MadLadSquad/YouyinWeb) and 
[hanzi-writer](https://github.com/chanind/hanzi-writer) easier. All scripts are located under the `server/` directory because that's where the character data is generated.

> 1. `process.py` - Reads the `make-me-a-hanzi` dictionary from the characters you have created and converts it to individual character files in `hanzi-writer` format under the `output` folder. Do not use directly, use `run.sh` instead
> 1. `run.sh` - Gets the character data from the meteor database, exports it to `tmp.json`, which is used as a `make-me-a-hanzi` dictionary file. It then runs `process.py`

Reviewer: UL
```suggestion
- `process.py` - Reads the `make-me-a-hanzi` dictionary from the characters you have created and converts it to individual character files in `hanzi-writer` format under the `output` folder. Do not use directly, use `run.sh` instead
- `run.sh` - Gets the character data from the meteor database, exports it to `tmp.json`, which is used as a `make-me-a-hanzi` dictionary file. It then runs `process.py`
```

### `youyin-dev-run.sh`
This script runs `./run.sh` for Youyin development. First, make sure you have the following repositories cloned in the same folder without renames:

> 1. make-me-a-hanzi-tool
> 1. [YouyinWeb](https://github.com/MadLadSquad/YouyinWeb)
> 1. [hanzi-writer-data-youyin](https://github.com/MadLadSquad/hanzi-writer-data-youyin)

Reviewer 2: Numbering
```suggestion
1. make-me-a-hanzi-tool
2. [YouyinWeb](https://github.com/MadLadSquad/YouyinWeb)
3. [hanzi-writer-data-youyin](https://github.com/MadLadSquad/hanzi-writer-data-youyin)
```

> 1. make-me-a-hanzi-tool
> 1. [YouyinWeb](https://github.com/MadLadSquad/YouyinWeb)
> 1. [hanzi-writer-data-youyin](https://github.com/MadLadSquad/hanzi-writer-data-youyin)

Reviewer: Unordered list
```suggestion
- make-me-a-hanzi-tool
- [YouyinWeb](https://github.com/MadLadSquad/YouyinWeb)
- [hanzi-writer-data-youyin](https://github.com/MadLadSquad/hanzi-writer-data-youyin)
```

Create a symbolic link so that `hanzi-writer-data-youyin/data/` points to `YouyinWeb/data/`: `ln -s YouyinWeb/data/ hanzi-writer-data-youyin/data/`

Next, edit `YouyinWeb/index.js` and change the `window.CHARACTER_FETCH_URL` string to `"http://0.0.0.0:8080/data/"`

With this setup, you can test characters. Now, every time you have finished a batch of characters, go into the `make-me-a-hanzi-tool/server` directory and run `./youyin-dev-run.sh`. It will:

> 1. Create `hanzi-writer` character files, like `run.sh`
> 1. Copy them to `hanzi-writer-data-youyin/data`
> 1. Go into `YouyinWeb` and run `./run.sh` to start the server.
> 1. Finally, open Youyin locally on `http://0.0.0.0:8080` and try creating character cards with the new characters.

Reviewer 2: Numbering
```suggestion
1. Create `hanzi-writer` character files, like `run.sh`
2. Copy them to `hanzi-writer-data-youyin/data`
3. Go into `YouyinWeb` and run `./run.sh` to start the server.
4. Finally, open Youyin locally on `http://0.0.0.0:8080` and try creating character cards with the new characters.
```

> 1. Finally, open Youyin locally on `http://0.0.0.0:8080` and try creating character cards with the new characters.

Reviewer: Unordered list makes more sense here
```suggestion
- Create `hanzi-writer` character files, like `run.sh`
- Copy them to `hanzi-writer-data-youyin/data`
- Go into `YouyinWeb` and run `./run.sh` to start the server.
- Finally, open Youyin locally on `http://0.0.0.0:8080` and try creating character cards with the new characters.
```

> Once all characters are working and generated correctly, simply got to the `hanzi-writer-data-youyin` directory, commit and push the changes.

Reviewer: Typo
```suggestion
Once all characters are working and generated correctly, simply go to the `hanzi-writer-data-youyin` directory, commit and push the changes.
```
